acceptor.go

```diff
@@ -228,6 +233,7 @@ func (a *Acceptor) invalidMessage(msg *bytes.Buffer, err error) {
 func (a *Acceptor) handleConnection(netConn net.Conn) {
 	defer func() {
 		if err := recover(); err != nil {
+			fmt.Println("asdqwe", a.globalLog)
```

Comment:
Do we need this?

Reply:
i mistakenly left that in... ...thank you for catching it!